backend/sozisel/lib/sozisel/events.ex

```diff
@@ -0,0 +1,118 @@
+defmodule Sozisel.Model.Events do
+  @moduledoc """
+  The Events context.
+  """
+
+  import Ecto.Query, warn: false
+  alias Sozisel.Repo
+
+  alias Sozisel.Model.Events.Event
+
+  @doc """
```

Comment:
Can you please remove those auto generated comments? It takes a lot of space and does not help at all. 

Add documentation if you think that function is quite complex and needs explanation.